src/robot/api/__init__.py

```diff
@@ -103,6 +103,7 @@
     TestSuiteBuilder as TestSuiteBuilder,
     TypeInfo as TypeInfo,
 )
+from robot.utils import Secret as Secret
```

Comment:
I don't think we should expose `Secret` directly via `robot.api` because that module is getting pretty crowded. Better to expose it only via `robot.api.types`.